### PR TITLE
Remove COW Entries

### PR DESCRIPTION
**Pull request by Maithem**
## Overview
Code clean up. Removed unused code.

### CODE REVIEW

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **d8e4ca0**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #1255 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/E5JxJJeW9HmVsB5J29SPnXGQKsnRY8KA)

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **2d30bfd**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #1255 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/XHk6R03Nc1LTFxxaLMB6AZQyfsEqVakm)

File: runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractContextStreamView.java [hunk 1]
```diff
@@ -301,13 +300,6 @@ protected void updatePointer(final ILogData data) {
     protected boolean processEntryForContext(final ILogData data) {
         if (data != null) {
             final Object payload = data.getPayload(runtime);
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractContextStreamView.java]: MAJOR Remove this useless assignment to local variable "payload". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "payload" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

File: runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java [hunk 2]
```diff
@@ -113,7 +113,6 @@ public boolean isMutation(UUID stream) {
         // Base Messages
         NOP(0, LogEntry.class),
         SMR(1, SMREntry.class),
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java]: MINOR Remove this use of "SMREntry"; it is deprecated. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java [hunk 3]
```diff
@@ -113,7 +113,6 @@ public boolean isMutation(UUID stream) {
...
-        STREAM_COW(4, StreamCOWEntry.class),
         MULTIOBJSMR(7, MultiObjectSMREntry.class),
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java]: MINOR Remove this use of "MultiObjectSMREntry"; it is deprecated. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java [hunk 4]
```diff
@@ -113,7 +113,6 @@ public boolean isMutation(UUID stream) {
...
         MULTISMR(8, MultiSMREntry.class),
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java]: MINOR Remove this use of "MultiSMREntry"; it is deprecated. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by corfu-repo** [comment]
SonarQube analysis reported 27 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 21 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ObjectsView.java#L63](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L63): Remove usage of generic wildcard type. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)
1. ![MAJOR][MAJOR] [LogEntry.java#L76](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java#L76): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MAJOR][MAJOR] [LogEntry.java#L107](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java#L107): Remove this unused method parameter "stream". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [ObjectsView.java#L3](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L3): Use classes from the Java API instead of Sun classes. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1191)
1. ![MAJOR][MAJOR] [AbstractContextStreamView.java#L324](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractContextStreamView.java#L324): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MINOR][MINOR] [ObjectsView.java#L3](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L3): Remove this unused import 'com.sun.xml.internal.bind.v2.TODO'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [ObjectsView.java#L14](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L14): Remove this unused import 'lombok.NonNull'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [ObjectsView.java#L24](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L24): Remove this unused import 'org.corfudb.runtime.object.CorfuCompileWrapperBuilder'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [ObjectsView.java#L30](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L30): Remove this unused import 'org.corfudb.runtime.view.stream.IStreamView'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [ObjectsView.java#L31](https://github.com/CorfuDB/CorfuDB/blob/10431922b431e3f9987a66a4fef0e602c788e560/runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java#L31): Remove this unused import 'org.corfudb.util.serializer.Serializers'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**